core/src/scss/utilities/mixins/pull-quote/_pull-quote.scss

```diff
@@ -0,0 +1,64 @@
+@charset "UTF-8";
+
+///
+/// Pull Quote Styling.
+///
+/// @name pull-quote
+///
+/// @param {variable} $side - Default side for the pull-quote.
+/// @param {variable} $border-color - Default color for the border.
+/// @param {variable} $mobile-background-color - Default background color for mobile width.
+/// @param {variable} $background-color - Default background color for desktop width.
+///
+/// @group mixin
+//
+
+@mixin pull-quote(
```

Comment:
Hmmm.... I like how you considered the mixin approach which allows for some options, but after working with them for a while, I'm now trying to avoid creating mixins with too many nesting elements as that tends to create more complexity than I would like. Could I convince you to add these styles to under the class name? 😄

Reply:
@yvonnetangsu ,
Thank you for that feedback. 

Yes! I wasn't sure the best approach for making this happen.